.github/workflows/release.yml

```diff
@@ -0,0 +1,25 @@
+on:
+  push:
+    tags:
+      - 'v*'
+
+name: Create Release
+
+jobs:
+  build:
+    name: Create Release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Create Release
+        id: create_release
+        uses: actions/create-release@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          tag_name: ${{ github.ref }}
+          release_name: Release ${{ github.ref }}
+          body_path: ./changelist.md
```

Comment:
Can we name this file `CHANGELOG.md`? I'm not really sure why but convention is to make the filename all caps like you're yelling at someone.

Reply:
sounds good and powerful, I'll change that to all caps.